jetty-websocket/websocket-util-server/src/main/java/org/eclipse/jetty/websocket/util/server/WebSocketUpgradeFilter.java

```diff
@@ -78,34 +77,40 @@
     private static final Logger LOG = LoggerFactory.getLogger(WebSocketUpgradeFilter.class);
     private static final AutoLock LOCK = new AutoLock();
 
+    /**
+     * The init parameter name used to define {@link ServletContext} attribute used to share the {@link WebSocketMapping}.
+     */
+    public static final String MAPPING_ATTRIBUTE_INIT_PARAM = "org.eclipse.jetty.websocket.util.server.internal.WebSocketMapping.key";
```

Comment:
This is the name of the init parameter used to define which `ServletContext` attribute to use to share the `WebSocketMapping`.

I am not really sure what use anyone would have for this as if they set it to anything else they would need to manually add their mappings to the `WebSocketMapping` which is an internal class.

I also don't really like the name `org.eclipse.jetty.websocket.util.server.internal.WebSocketMapping.key`, I think this could simplify this to just `WebSocketMapping`.

@gregw what do you think?